.repo-metadata.json

```diff
@@ -1,19 +1,19 @@
 {
   "api_shortname": "pubsub",
   "name_pretty": "Cloud Pub/Sub",
-  "api_reference": "https://cloud.google.com/pubsub/",
   "product_documentation": "https://cloud.google.com/pubsub/docs/",
-  "client_documentation": "https://cloud.google.com/java/docs/reference/google-cloud-pubsub/latest/history",
   "api_description": "is designed to provide reliable, many-to-many, asynchronous messaging between applications. Publisher applications can send messages to a topic and other applications can subscribe to that topic to receive the messages. By decoupling senders and receivers, Google Cloud Pub/Sub allows developers to communicate between independently written applications.",
-  "issue_tracker": "https://issuetracker.google.com/savedsearches/559741",
+  "client_documentation": "https://cloud.google.com/java/docs/reference/google-cloud-pubsub/latest/history",
```

Comment:
Possibly change to "https://cloud.google.com/java/docs/reference/google-cloud-pubsub/latest" to link to the overview page instead of the version history.